plugins/scenario/worker/Worker.ts

```diff
@@ -62,6 +63,11 @@ export async function run<T, U, R>({ bases, config, worker }: WorkerData) {
   for (const base of bases) {
     const world = new World(hreForBase(base), base), dm = world.deploymentManager;
     const delta = await dm.runDeployScript({ allMissing: true });
+
+    // Reading pending proposals and storing them in the cache for all scenarios
+    const proposals = await getAllPendingProposals(world);
+    dm.cache.storeCache(getProposalCacheId(base.network), proposals);
```

Comment:
The cache is stored across networks, so not sure we would want to do this without accounting for forking. Also, the plugin code is meant to be agnostic to Compound-specific stuff, so we wouldn't want it to be aware of proposals, right?